spec/features/delete_draft_collection_spec.rb

```diff
@@ -16,7 +16,9 @@
       visit dashboard_path
 
       accept_confirm do
-        click_link "Delete #{collection.name}"
+        within '#your-collections' do
```

Comment:
Would `within_section 'Your collections'` work here?

Reply:
It doesn't work, perhaps because the "+ Create a new collection" button is in the <section><header> tag too

Reply:
Yeah, I was afraid of that.